static/src/javascripts/projects/commercial/modules/user-features.js

```diff
@@ -21,7 +21,11 @@ const persistResponse = (JsonResponse: () => void) => {
     expiryDate.setDate(expiryDate.getDate() + 1);
     addCookie(USER_FEATURES_EXPIRY_COOKIE, expiryDate.getTime().toString());
     addCookie(PAYING_MEMBER_COOKIE, !JsonResponse.adblockMessage);
-    addCookie(AD_FREE_USER_COOKIE, JsonResponse.adFree);
+    if (JsonResponse.adFree) {
+        addCookie(AD_FREE_USER_COOKIE, JsonResponse.adFree, 1);
+    } else {
+        removeCookie(AD_FREE_USER_COOKIE);
```

Comment:
I think this removal causes multiple extra requests to the Members Data API /features endpoint via the logic in line 47. See https://manage.fastly.com/stats/service/3oeL55htke1szmt67tNxEq/in/all/from/1497312000000/to/1497442440000/by/minute